Cargo.toml

```diff
@@ -15,6 +15,8 @@ categories = ["development-tools"]
 [dependencies]
 nom = "7.0.0"
 unicode_categories = "0.1.1"
+lazy_static = "1"
```

Comment:
The most recent recommendation is to migrate from `lazy_static` to [`once_cell`](https://github.com/matklad/once_cell)

Reply:
@shssoichiro Done! :)